reactive/etcd.py

```diff
@@ -25,6 +25,7 @@
 from charmhelpers.core import host
 from charmhelpers.fetch import apt_update
 from charmhelpers.fetch import apt_install
+from charmhelpers.contrib.charmsupport import nrpe
```

Comment:
No, it's in charmhelpers, which is provided by hmmm the basic layer I think ? I'm not sure I understand your question.